Viper_project/Viper_project/Services/NetworkService.swift

```diff
@@ -0,0 +1,71 @@
+// NetworkService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Сетевой слой
+final class NetworkService: NetworkServiceProtocol {
+    // MARK: - Public methods
+
+    func fetchCountries(_ completion: @escaping CountriesHandler) {
+        let countries = Countries.all
+        if !countries.isEmpty {
+            completion(.success(countries))
+        } else {
+            let error = NSError(domain: Constants.errorDomainValue, code: 1)
+            completion(.failure(error))
+        }
+    }
+
+    func fetchHotels(id: String, _ completion: @escaping HotelsHandler) {
+        switch id {
+        case Constants.vietnamId:
+            completion(.success(Hotels.vietnam))
+        case Constants.turkeyId:
+            completion(.success(Hotels.turkey))
+        case Constants.thaiId:
+            completion(.success(Hotels.thai))
+        case Constants.baliId:
+            completion(.success(Hotels.bali))
+        case Constants.kzId:
+            completion(.success(Hotels.kazakhstan))
+        default:
+            let error = NSError(domain: Constants.errorDomainValue, code: 0)
+            completion(.failure(error))
+        }
+    }
+}
+
+/// Моковые страны и отели
+private extension NetworkService {
+    enum Countries {
```

Comment:
```suggestion
    enum Countries {
```
это все должно быть в константах Свифтгена
глянь как у ребят